vizro-core/src/vizro/models/_components/container.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, List, Literal
+
+from dash import html
+
+try:
+    from pydantic.v1 import Field, validator
+except ImportError:  # pragma: no cov
+    from pydantic import Field, validator
+
+from vizro.models import VizroBaseModel
+from vizro.models._models_utils import (
+    _assign_component_grid_area,
+    _create_component_container,
+    _log_call,
+    set_components,
+    set_layout,
+)
+from vizro.models.types import ComponentType
+
+if TYPE_CHECKING:
+    from vizro.models import Layout
+
+
+class Container(VizroBaseModel):
+    """Html.Div that can wrap other `Page.components`.
+
+    Args:
+        type (Literal["container"]): Defaults to `"container"`.
+        components (List[ComponentType]): See [ComponentType][vizro.models.types.ComponentType]. At least one component
+            has to be provided.
+        title (str): Title to be displayed.
+        layout (Layout): Layout to place components in. Defaults to `None`.
+
+    Raises:
+        ValueError: If number of page and grid components is not the same
```

Comment:
@maxschulz-COL should we get rid of this across all models (not all functions)? There's any number of reasons a pydantic model could fail validation, and keeping this up to date seems like a hassle (it's already not up to date here or in `Page` since it doesn't mention the "at least one component per page" restriction).

Reply:
Aligned with @maxschulz-COL that we'll remove it 👍

Reply:
Sorry forgot about this comment. Yes I agree, it seems in line with the  [google style guidelines](https://google.github.io/styleguide/pyguide.html#doc-function-raises):

> You should not document exceptions that get raised if the API specified in the docstring is violated